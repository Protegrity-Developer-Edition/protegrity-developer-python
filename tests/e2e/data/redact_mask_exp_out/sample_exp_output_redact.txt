My name is [PERSON], and I live in [CITY]. My Social Security number is [SSN], and I am [AGE]. You can reach me on my cell at [PHONE] - that's a [STATE] number. The username I use to access my account is [USERNAME].