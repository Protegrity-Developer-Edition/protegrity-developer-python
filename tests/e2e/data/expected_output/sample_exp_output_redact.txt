The customer reported their card number as [CCN], but they might need to check it again. During the verification process, we noticed [CCN] was submitted multiple times by mistake. Please ensure you replace the incorrect input [DATE] with valid data. I think [DATE] was listed in a previous entry, but double-check just in case. For reference, [DATE] seems to follow an old format; update as required.